lib/NodeFactory/NodeFactory.php

```diff
@@ -75,6 +77,9 @@ public function createSpanNode(Parser $parser, $span): SpanNode;
 
     public function createDefinitionListNode(DefinitionList $definitionList): DefinitionListNode;
 
+    /** @param FieldOption[] $items */
+    public function createFieldListNode(array $items): FieldListNode;
```

Comment:
This should be documented in the UPGRADE.md I suppose, as it is a breaking change